File: quickstart/csharp/voice/outgoing-call/outgoing-call.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;
using Twilio.Types;

namespace YourNewConsoleApp
{
    class Program
    {
        static void Main(string[] args)
        {
           // Find your Account Sid and Auth Token at twilio.com/console
           const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
           const string authToken = "your_auth_token";
           TwilioClient.Init(accountSid, authToken);

           var to = new PhoneNumber("+14155551212");
           var from = new PhoneNumber("+15017122661");
           var call = CallResource.Create(to, from,
               url: new Uri("http://demo.twilio.com/docs/voice.xml"));

           Console.WriteLine(call.Sid);
        }
    }
}

File: quickstart/ruby/voice/example-1-make-call/outgoing_call.5.x.rb
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# set up a client to talk to the Twilio REST API
@client = Twilio::REST::Client.new(account_sid, auth_token)

call = @client.calls.create(
    to: "+15558675310",
    from: "+15017122661",
    url: "http://demo.twilio.com/docs/voice.xml")
puts call.to